libs/sdk-backend-tiger/tests/reference_workspace/create_test_workspaces.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+// (C) 2022 GoodData Corporation
+
+fs = require("fs");
+const path = require("path");
+
+require("dotenv").config();
+
+const {
+    createTigerWorkspaceWithPrefix,
+    defaultScanPublishPDMForDatasource,
+    setTigerWorkspaceLayout,
+} = require("@gooddata/fixtures");
+const { log } = require("@gooddata/fixtures/logger.js");
+
+const UISDK_EMPTY_WORKSPACE_PREFIX = "Test empty UISDK reference workspace";
+
+const goodSalesLayoutFilePath = path.resolve(`./tests/reference_workspace/fixtures/layout/layout.json`);
+
+async function setWorkspaceFixtures(workspaceId, fixtureFile, token, host, backend) {
```

Comment:
I would like to consolidate these function from all repos and create universal implementation placed in share gdc-fixtures or in UI SDK and re-use it everywhere